client/components/inline-support-link/index.jsx

```diff
@@ -68,6 +65,15 @@ class InlineSupportLink extends Component {
 		}
 	}
 
+	prefetchAlternates = () => {
+		const { supportPostId } = this.props;
+		const queryClient = new QueryClient();
```

Comment:
This prefetches the query into a brand new `QueryClient` instance and then garbage collects it, doesn't it? The `useSupportArticleAlternatesQuery` will never see the prefetched data. You need to get the global query client with `useQueryClient()`.

If it was up to me, I would give up on the preloading if possible. Fetch only after the modal is opened.

Reply:
You're right. Also, I did think about removing the prefetching, so I'm happy you brought it up. 

Removed prefetching in ae62c008911be956431c519671c21837e286a799.